google/cloud/pubsub/internal/streaming_subscription_batch_source.h

```diff
@@ -135,6 +135,8 @@ class StreamingSubscriptionBatchSource
   std::chrono::seconds const max_deadline_time_;
 
   std::mutex mu_;
+  // A number only used once to identify modack requests for tracing.
```

Comment:
I'm not sure what the "only [used] once" is supposed to convey.

Reply:
That it is unique to the modack request. But this irrelevant now.